test.js

```diff
@@ -122,9 +132,6 @@ describe('styled-jsx-plugin-sass', () => {
 
     assert.equal(
       plugin(file.toString(), {
-        sassOptions: {
```

Comment:
we should keep this I guess, tests should still pass with the old setup

Reply:
No, that test explicitly uses a relative import (i.e. starts with `./`). The expected behavior is that it will resolve relative to the current file.

Reply:
In other words. The test as is, is verifying an undesirable behavior. You shouldn't have to add to `includePaths` if you're explicitly using `./` to signal a relative path.